db/migrate/20210702143800_add_length_constraint_to_name_to_keywords.rb

```diff
@@ -0,0 +1,8 @@
+class AddLengthConstraintToNameToKeywords < ActiveRecord::Migration[6.1]
+  def change
+    add_check_constraint :keywords, 'char_length(name) <= 255', name: 'char_length_name'
+
+    execute("ALTER TABLE keywords ALTER COLUMN created_at SET DEFAULT CURRENT_TIMESTAMP")
```

Comment:
@malparty I guess this part has nothing to do with `AddLengthConstraintToNameToKeywords`, so better create a seperate migration. Also, it should be reversible migration, so we would need to define both up and down methods in this case.

```
   class SetDefaultValueForKeywordTimestamps < ActiveRecord::Migration[6.1]
      def up
        execute("ALTER TABLE keywords ALTER COLUMN created_at SET DEFAULT CURRENT_TIMESTAMP")
     end

     def down
       ## Reverse this migration
     end
```

Reply:
Smart remark! Thanks, it's done ;-)